src/components/LeaderListItem/index.jsx

```diff
@@ -14,12 +14,18 @@ const LeaderListItem = props => {
       <span className="placement">{props.placement}</span>
       <Avatar size={40} src={props.image} />
 
-      <span className="name">
-        <Link to={'/profile/' + props.uuid}>{props.name}</Link>
+      <span className="column left">
+        <div className="name">
+          <Link to={'/profile/' + props.uuid}>{props.name}</Link>
+        </div>
+        <div className="rank mobile">{getRank(props.exp)}</div>
       </span>
 
-      <span className="rank">{getRank(props.exp)}</span>
-      <span className="exp">{props.exp} points</span>
+      <span className="rank desktop">{getRank(props.exp)}</span>
+
```

Comment:
huh does running lint:fix not remove the extra blank lines above and below span?

Reply:
Nope. It was in the original source but I removed the extra lines.